backend/src/schema/resolvers/helpers/generateNonce.js

```diff
@@ -0,0 +1,4 @@
+import uuid from 'uuid/v4'
+export default function generateNonce() {
+  return uuid().substring(0, 6)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-27T22:44:16Z" title="Saturday, September 28th 2019, 12:44:16 am +02:00">Sep 28, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,4 @@
+import uuid from 'uuid/v4'
+export default function generateNonce() {
+  return uuid().substring(0, 6)
```


@datenbrei you wanted to know how the nonce gets generated